added turbulence and computational mechanics courses (#94)
anovak-lab/group_resources: academics/courses.md

## TAM

### Computational Mechanics (TAM 470)

This course is a general numerical analysis course with a strong engineering focus. It covers key topics including interpolation, numerical differentiation and integration, numerical methods for ordinary differential equations, and numerical methods for partial differential equations, with an emphasis on finite difference methods and brief introductions to finite volume and finite element methods. Issues of accuracy and stability of numerical schemes are also addressed. The course content is well structured and comprehensive, and the instructor is excellent. The workload consists of 12 assignments of moderate difficulty, two coding projects, three quizzes, and a final exam. The exams are relatively easy, and the assignments require only basic Python knowledge. Overall, this is a very good course and one that I would highly recommend.

### Turbulence (TAM 538)

> This course provides a theoretical introduction to turbulence and requires a prior course in fluid dynamics. It focuses primarily on the Reynolds-averaged Navier–Stokes (RANS) equations. Topics include statistical tools used in turbulence, mean flow equations, free shear flows, wall-bounded flows, and proper orthogonal decomposition. The course does not cover computational turbulence models. The course grade is based on three exams and a final project, with the project evaluated according to its potential to lead to a journal publication. I do not recommend this course due to the absence of computational turbulence modeling, which is essential for CFD. Additionally, the lack of regular assignments makes the material more difficult to grasp and places excessive pressure on the exams.

Reviewer: ```suggestion
This course provides a theoretical introduction to turbulence and requires a prior course in fluid dynamics. It focuses primarily on the Reynolds-averaged Navier–Stokes (RANS) equations. Topics include statistical tools used in turbulence, mean flow equations, free shear flows, wall-bounded flows, and proper orthogonal decomposition. The course does not cover computational turbulence models. The course grade is based on three exams and a final project, with the project evaluated according to its potential to lead to a journal publication. This course lacks computational turbulence modeling, which is essential for CFD. Additionally, the lack of regular assignments makes the material more difficult to grasp and places excessive pressure on the exams.
```

Reviewer: "I" could mean me, so just making it more generic :)
